README.md

```diff
@@ -29,14 +29,10 @@ Glass please download the source archive from the website instead:
 
 https://looking-glass.io/downloads
 
-Please also be sure to see the following files for more information
-Note: The `README.md` files are slowly being deprecated from this project in
-favor of the wiki at https://looking-glass.io/wiki, and as such the
-information in these files may be dated.
-
-* [client/README.md](client/README.md)
-* [host/README.md](host/README.md)
-* [module/README.md](module/README.md)
+Further documentation can be found in the `/doc` directory.
+
+You may view this locally as HTML by running `make html` with `python3-sphinx`
+installed, or online at https://looking-glass.io/docsurlhere
```

Comment:
Replace this URL with the hosted version when it comes online

Reply:
On second thought, should this portion be removed for now? My thinking is, since this is the bleeding edge repo, the README can be unfinished until next release, but you might prefer it remain valid because it recieves high traffic.


Other than this, the PR is ready